restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -533,35 +531,64 @@ private void invite(final Object message) throws IOException, NumberParseExcepti
 
                 if (isWebRTC(request)) {
```

Comment:
@abdulazizali77 if this is a WebRTC request it will be proxied to PSTN (with RMS in the path) so the `IExtensionCreateCallRequest` should be for `CreateCallType.PSTN`

Also, the extension execution here is duplicate and we can execute only once for both WebRTC or regular sip calls